src/Application.php

```diff
@@ -84,6 +86,8 @@ public function __construct(string $workshopTitle, string $diConfigFile)
 
         $this->workshopTitle = $workshopTitle;
         $this->diConfigFile = $diConfigFile;
+
+        set_error_handler([$this, 'handleInternalError']);
```

Comment:
This method seems empty and we then call `set_error_handler` again in `configure` - is this a refactoring artefact ?

Reply:
yeah just shit I need to delete 👀